### PR TITLE
feat(camel-plugin): Add 'Endpoint (in/out)' view into the camel plugin

### LINKED ISSUE #282: Camel - Endpoint Statistics view
`/camel/endpoints-statistics`

Screenshot - TBD. Let's investigate whether we need this view for Camel 3+ or not.

### PR DESCRIPTION
**Pull request by mmelko**
Add component test to cover rendering EndpointStats component.
closes #282 
Screenshot 2023-06-14 at 15 18 31
Filtering:
Screenshot 2023-06-14 at 15 18 48

### CODE REVIEW

File: packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts [hunk 1]
```diff
@@ -280,3 +292,13 @@ function parseMessagesFromXml(pDoc: XMLDocument): MessageData[] {
 
   return messagesData
 }
+
+export async function getEndpointStatistics(node: MBeanNode) {
+  let stats: EndpointStatistics[] = []
+  const registry = getDefaultRuntimeEndpointRegistry(node)
+  if (registry) {
+    const res = await jolokiaService.execute(registry.objectName ?? '', 'endpointStatistics()')
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts:300]: if `objectName` is absent it doesn't make sense to invoke the method. Rather shouldn't we guard it based on its presence and if it's absent something else should be done, like throwing or logging error?

**Review by tadayosi — APPROVED** [review]
Overall, looks great. There's one minor comment.

File: packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts (hunk 1 above)
Comment (mmelko) [reply packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts]: updated

File: packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts [hunk 2]
```diff
@@ -280,3 +292,17 @@ function parseMessagesFromXml(pDoc: XMLDocument): MessageData[] {
...
+export async function getEndpointStatistics(node: MBeanNode) {
+  let stats: EndpointStatistics[] = []
+  const registry = getDefaultRuntimeEndpointRegistry(node)
+  if (registry) {
+    if (registry.objectName) {
+      const res = await jolokiaService.execute(registry.objectName ?? '', 'endpointStatistics()')
+      stats = Object.values(res as { [key: string]: EndpointStatistics })
+    } else {
+      log.error('Error with the retrieving registry')
+    }
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts:305]: so, now the code just ignores if `registry` is absent. Is it what we expect?

Also, now you already guard with `if (regitry.objectName)` so `registry.objectName ?? ''` doesn't make sense as you can assume it's present.

**Review by tadayosi — APPROVED** [review]

**Pull request merged**